workspace/solution.go

```diff
@@ -67,17 +69,15 @@ func (s *Solution) Write(dir string) error {
 	if err != nil {
 		return err
 	}
-
-	path := filepath.Join(dir, solutionFilename)
-
-	// Hack because ioutil.WriteFile fails on hidden files
-	visibility.ShowFile(path)
-
-	if err := ioutil.WriteFile(path, b, os.FileMode(0600)); err != nil {
+	if err = createIgnoreSubdir(dir); err != nil {
```

Comment:
I'd probably inline this. I know `createIgnoreSubdir` is used in a couple of places, but it doesn't feel like an actual abstraction to me, just an accidental bit of logic that happens to have to happen twice.

Reply:
Agreed. As you mentioned below, using `MkdirAll` makes this just 2 lines.